Adyen/Core/APIClient/Environment.swift

```diff
@@ -43,6 +43,12 @@ public struct Environment: AnyAPIEnvironment {
 
     /// Adyen's India live  environment.
     public static let liveIndia = Environment(baseURL: URL(string: "https://checkoutshopper-live-in.adyen.com/")!)
+    
+    /// Determines if the environment is one of Adyen's live environments.
+    @_spi(AdyenInternal)
+    public var isLive: Bool {
+        baseURL.absoluteString.hasPrefix("https://checkoutshopper-live")
```

Comment:
what about having a prefix property `static let liveEnvironmentPrefix: String` that we compose the live environments with - so a change on the environment urls does not accidentally break this check.

```suggestion
        baseURL.absoluteString.hasPrefix(Self.liveEnvironmentPrefix)
```

Reply:
Nice suggestion. but I'd imagine a change on live url could mean a bigger issue than this SDK :D

But a separate PR to encompass all the common part of the domain path could still be nice